packages/nextjs/app/api/og/route.tsx

```diff
@@ -0,0 +1,269 @@
+              />
+            </svg>
+
+            {/* Avatar */}
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            {avatarUrl ? (
+              <img
+                src={avatarUrl || ""}
+                width={200}
+                height={200}
+                alt="Avatar"
+                style={{
+                  margin: 0,
+                  marginTop: 40,
+                  borderRadius: "50%",
+                  borderWidth: 5,
+                  borderColor: "white",
+                  borderStyle: "solid",
+                }}
+              />
+            ) : (
+              <div
+                style={{
+                  margin: 0,
+                  marginTop: 40,
+                  display: "flex",
+                  overflow: "hidden",
+                  width: 117,
+                  height: 117,
+                  borderRadius: "50%",
+                  borderWidth: 5,
+                  borderColor: "white",
+                  borderStyle: "solid",
+                }}
+              >
+                <img
+                  alt={address}
+                  src={blo(address as `0x${string}`, PUNK_SIZE * computedScale)}
+                  style={{
+                    position: "absolute",
+                    opacity: 0.5,
+                    inset: 0,
+                  }}
+                />
+                <img
+                  width={`${2400 * PUNK_SIZE_RATIO * computedScale}`}
+                  height={`${2400 * PUNK_SIZE_RATIO * computedScale}`}
+                  src={imageData}
+                  alt="Punks Image"
+                  style={{
+                    position: "absolute",
+                    top: `${-PUNK_SIZE * backgroundPositionY * computedScale}px`,
+                    left: `${-PUNK_SIZE * backgroundPositionX * computedScale}px`,
+                    imageRendering: "pixelated",
+                  }}
+                />
+              </div>
+            )}
+
+            {/* Address or ENS Name Text */}
+            <div
+              style={{
+                marginTop: 10,
+                fontSize: 44,
+                fontWeight: "bold",
+                color: colors.text,
+                textAlign: "center",
+              }}
+            >
+              {ensName || `${checksumAddress.slice(0, 6)}...${checksumAddress.slice(-6)}`}
+            </div>
+
+            <div
+              style={{
+                marginTop: 5,
+                marginBottom: 16,
+                display: "flex",
+                fontSize: 36,
+                fontWeight: "bold",
+                color: colors.text,
+                textAlign: "center",
+              }}
+            >
+              Challenges Completed: {challenges.length}
```

Comment:
I think these two _compete_ too much
![image](https://github.com/user-attachments/assets/991f1e55-912a-457e-b790-59adf001ad64)

Maybe we need one of the two to be bigger / smaller or bold / light?

Reply:
I've adjusted the font sizes and colors, it doesn't seem like font weight is working though

Reply:
This looks great!! <3

Will do a final pass tomorrow.

Thanks Hunter!